src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    public class ManagementHubOptionsSetup : IConfigureOptions<HubOptions>
+    {
+        internal static TimeSpan DefaultHandshakeTimeout => TimeSpan.FromSeconds(15);
+
+        internal static TimeSpan DefaultKeepAliveInterval => TimeSpan.FromSeconds(15);
+
+        internal const int DefaultMaximumMessageSize = 32 * 1024;
+
+        internal const int DefaultStreamBufferCapacity = 10;
+
+        private readonly List<string> _defaultProtocols = new List<string>();
+
+        public ManagementHubOptionsSetup(IEnumerable<IHubProtocol> protocols)
+        {
+            foreach (var hubProtocol in protocols)
+            {
+                if (hubProtocol.GetType().CustomAttributes.Where(a => a.AttributeType.FullName == "Microsoft.AspNetCore.SignalR.Internal.NonDefaultHubProtocolAttribute").Any())
```

Comment:
you don't expose DI, so you don't need this?

Reply:
How is this resolved?

Reply:
failed to git push, updated